src/tr/mangaruhu/src/eu/kanade/tachiyomi/extension/tr/mangaruhu/MangaRuhu.kt

```diff
@@ -0,0 +1,14 @@
+package eu.kanade.tachiyomi.extension.tr.mangaruhu
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class MangaRuhu : Madara(
+    "Manga Ruhu",
+    "https://mangaruhu.com",
+    "tr",
+    SimpleDateFormat("dd/MM/yyyy", Locale.ENGLISH),
```

Comment:
Date format doesn't seem to match site. Is the API returning a different format?

Try:
```suggestion
    SimpleDateFormat("d MMMM yyyy", Locale("tr")),
```